src/com/tony/server/response/LogResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.tony.server.response;
+
+import com.tony.server.Logger;
+
+import java.util.HashMap;
+
+public class LogResponse extends Response{
+    private final Logger logger;
+
+    public byte[] respond(){
+        setBody(logger.getLog());
+        return ResponseBuilder.buildResponse(getStatusLine(),
+                getHeaders(), getBody()).getBytes();
+    } 
+    
+    public LogResponse(Logger logger) {
+        this.logger = logger;
+        setStatusLine("HTTP/1.1 200 OK");
```

Comment:
Do you think it's better to do what you said, like setting the Status-Codes and Response-Phrases constants in the Response's constructor and using ResponseBuilder or to have a constant for each Status Line with all of those things instead?